cognite/powerops/custom_modules/power_model_v1/data_models/containers/SHOPResult.container.yaml

```diff
@@ -0,0 +1,108 @@
+space: '{{powerops_models}}'
+externalId: SHOPResult
+name: SHOPResult
+usedFor: node
+properties:
+  shopTimeseries1:
+    type:
+      list: false
```

Comment:
General FDM question: Is it so that we don't need to refer to a container for properties (in a type/view) that are lists of "complex types"?

Reply:
yes true. This is solved by using the `multi_edge_connection` with a source of the type `view`